### PR TITLE
fix(menubar): Add aria-selected and aria-activedescendant to menu bar items (fix #3911)

### LINKED ISSUE #3911: [BITV] 9.4.1.2/3.16 - Custom editing menubar should be implemented using role=toolbar and not role=menubar. (2)
Currently selected styles on selected text in the editing region ("div" with "contenteditable=true") are shown visually (light blue background) but they are not programmatically determinable via aria-selected attribute. aria-activedescendant is not used on toolbar styling elements. Also missing is appropriate scripted focus management for the style toolbar and for its drop-down submenus like headings, insert image, insert emoi). 

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v3-n16

</details>

- [ ] 9.4.1.2/10.16 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-bfbf07e5c3-v10-n16

### PR DESCRIPTION
**Pull request by juliushaertl**
### 📝 Summary

Fixes https://github.com/nextcloud/text/issues/3911

### 🏁 Checklist

- [ ] Code is properly formatted (`npm run lint` / `npm run stylelint` / `composer run cs:check`)
- [ ] [Sign-off message](https://probot.github.io/apps/dco/) is added to all commits
- [ ] [Tests](https://github.com/nextcloud/text#-testing-the-app) (unit, integration and/or end-to-end) passing and the changes are covered with tests
- [ ] Documentation ([README](https://github.com/nextcloud/text/blob/main/README.md) or [documentation](https://github.com/nextcloud/documentation/blob/master/admin_manual/configuration_server/text_configuration.rst#L2)) has been updated or is not required

### CODE REVIEW

File: src/components/Menu/ActionList.vue [hunk 1]
```diff
@@ -27,6 +27,7 @@
 		v-bind="state"
 		:container="menuIDSelector"
 		:aria-label="actionEntry.label"
+		:aria-activedescendant="!!currentChild"
```
Comment (susnux) [inline src/components/Menu/ActionList.vue:30]: This should not be a boolean value, but the ID of the currently active (focused) element.

**Review by skjnldsv — APPROVED** [review]

**Review by susnux — CHANGES_REQUESTED** [review]

File: src/components/Menu/ActionList.vue (hunk 1 above)
Comment (susnux) [reply src/components/Menu/ActionList.vue:30]: Meaning this should be set on the menu toggle with the value of the active action button in the popover

Comment (juliushaertl) [reply src/components/Menu/ActionList.vue:30]: Right, seems I mixed that up, will have a look

Comment (juliushaertl) [reply src/components/Menu/ActionList.vue]: Adjusted to have a unique id matching with the menu entries.

**Review by susnux — APPROVED** [review]

**Comment by JuliaKirschenheuter** [comment]
/backport 15ae86a33eace3fbb355e12ac980e7cfe5a8c9a9 to stable28

**Pull request merged**